input.json

```diff
@@ -4,7 +4,7 @@
     "measures": "uniques",
     "chart_type": "bar",
     "interval": "daily",
-    "interval_range": "10d",
+    "interval_range": "21d",
     "repeat": "f",
```

Comment:
repeat should now be false
It's a boolean, not string
Look at alerts for example